sdk/trace-shaded-deps/src/test/java/io/opentelemetry/sdk/trace/internal/SunMiscProhibitedSecurityManagerTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.trace.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.security.AccessControlException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class SunMiscProhibitedSecurityManagerTest {
+
+  @Test
+  public void checkPackageAccess_ProhibitsSunMisc() {
+    SunMiscProhibitedSecurityManager sm = new SunMiscProhibitedSecurityManager();
+    AccessControlException thrown =
+        Assertions.assertThrows(
```

Comment:
Looks like a junit assertion was able to sneak through.

Reply:
thanks, fixed